### PR TITLE
{chem} [ictce/7.1.2] Plumed 2.04 (REVIEW)

### PR DESCRIPTION
**Pull request by wpoely86**
@pforai updated plumed easyconfig, please test

### CODE REVIEW

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/ba2c5b6ef83e957597d4 for a full test report.

**Comment by ocaisa** [comment]
style review ok

**Comment by ocaisa** [comment]
Testing

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/927e8e04d8652e54fdf2 for a full test report.

**Comment by ocaisa** [comment]
Test passed for me too, good to merge I think

**Comment by ocaisa** [comment]
works for me

**Comment by boegel** [comment]
Thanks @wpoely86, and @ocaisa for the review!

File: easybuild/easyconfigs/p/plumed/plumed-2.0.4-ictce-7.1.2.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+name = 'plumed'
+version = '2.0.4'
+
+homepage = 'http://www.plumed-code.org'
+description = """PLUMED is an open source library for free energy calculations in molecular systems which
+works together with some of the most popular molecular dynamics engines. Free energy calculations can be 
+performed as a function of many order parameters with a particular  focus on biological problems, using
+state of the art methods such as metadynamics, umbrella sampling and Jarzynski-equation based steered MD.
+The software, written in C++, can be easily interfaced with both fortran and C/C++ codes.
+"""
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/p/plumed/plumed-2.0.4-ictce-7.1.2.eb]: single leading space before description lines helps readability, plz fix.

File: easybuild/easyconfigs/p/plumed/plumed-2.0.4-ictce-7.1.2.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+toolchain = {'name': 'ictce', 'version': '7.1.2'}
+
+# Download from http://www.plumed-code.org/get-it, google drive doesn't allow for clean download url
+source_urls = ['https://github.com/plumed/plumed2/archive/']
+sources = ['v2.0.4.tar.gz']
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/p/plumed/plumed-2.0.4-ictce-7.1.2.eb]: avoid hardwired version here, to assist with `--try-software-version` facility

**Comment by fgeorgatos** [comment]
too late ;-)

**Comment by boegel** [comment]
I can submit a follow-up PR to fix your remarks, sorry... Anything else?

**Pull request merged**